src/main/java/teammates/common/util/LogHelper.java

```diff
@@ -0,0 +1,128 @@
+
+import com.google.appengine.api.log.AppLogLine;
+import com.google.appengine.api.log.LogQuery;
+import com.google.appengine.api.log.LogServiceFactory;
+import com.google.appengine.api.log.RequestLogs;
+import com.google.appengine.api.log.LogService.LogLevel;
+import com.google.appengine.api.modules.ModulesService;
+import com.google.appengine.api.modules.ModulesServiceFactory;
+
+public class LogHelper {
+    /**
+     * 6 past versions to query, including the current version and its 5 preceding versions.
+     */
+    public static final int MAX_PAST_VERSIONS_TO_QUERY = 6;
+    
+    private LogQuery query;
+    
+    public LogHelper() {
+        query = LogQuery.Builder.withDefaults();
+    }
+    
+    /**
+     * Sets values for query.
+     * If versionsToQuery is null or empty, the current version with its 5 preceding versions will be used instead.
+     * 
+     * @param versionsToQuery 
+     * @param startTime
+     * @param endTime
+     * @param offset
+     */
+    public void setQuery(List<String> versionsToQuery, Long startTime, Long endTime, String offset) {
+        query.includeAppLogs(true);
+        query.batchSize(1000);
+        query.minLogLevel(LogLevel.INFO);
+        if (startTime != null) {
+            query.startTimeMillis(startTime);
+        }
+        if (endTime != null) {
+            query.endTimeMillis(endTime);
+        }
+        query.majorVersionIds(getVersionIdsForQuery(versionsToQuery));
+        if (offset != null && !offset.equals("null")) {
+            query.offset(offset);
+        }
+    }
+    
+    /**
+     * Selects versions for query. If versions are not specified, it will return 
+     * default versions used for query.
+     */
+    private List<String> getVersionIdsForQuery(List<String> versions) {
+        boolean isVersionSpecifiedInRequest = (versions != null && !versions.isEmpty());
+        if (isVersionSpecifiedInRequest) {
+            return versions;
+        }
+        return getDefaultVersionIdsForQuery();
+    }
+    
+    /**
+     * Gets a list of versions, including the current version and 5 preceding versions (if available).
+     * @return a list of default versions for query.
+     */
+    private List<String> getDefaultVersionIdsForQuery() {
+        ModulesService modulesService = ModulesServiceFactory.getModulesService();
+        String[] versionList = (String[]) modulesService.getVersions(null).toArray(); // null == default module
+        String currentVersion = modulesService.getCurrentVersion();
+        int currentVersionIndex;
+        List<String> defaultVersions = new ArrayList<String>();
+        try {
+            currentVersionIndex = getCurrentVersionIndex(versionList, currentVersion);
+            defaultVersions = getNextFewVersions(versionList, currentVersionIndex);
+        } catch (InvalidParametersException e) {
+            defaultVersions.add(currentVersion);
+            e.printStackTrace();
```

Comment:
`e.printStackTrace()` is hardly helpful. We can either don't have it, or do a `log.*` instead, although I'm not sure which log level is appropriate here.